File: Pwr/Sem4/JezykiIParadygmatyProgramowania/Lista1/zad1/mainEuler.c
#include <stdio.h>
#include "euler.h"

int main()
{
    printf("%d\n", NWD(36, 8));
    printf("%d\n", minimalDivisor(15));
    printf("%d\n", Totient(11));

    Pair ans = diophantic(15, 21);
    printf("%d %d\n", ans.first, ans.second);

    return 0;
}

File: Pwr/Sem4/JezykiIParadygmatyProgramowania/Lista1/zad3/maineuler.go
package maineuler

import (
	"fmt"
	"euler"
)

func main() {
	fmt.Println("GCD 24 and 36 is", euler.GCD(24,36))
	fmt.Println("Minimal divisor of 91 is", euler.MinimalDivisor(91))
	fmt.Println("Totient of 11 is", euler.Totient(11))
	p := euler.Diophantic(15, 10)
	fmt.Println("Diophantic solution:", p.First, p.Second)
}